### PR TITLE
[14.0][FIX] repair_sale_order, keep shipping address 

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
Keep shipping address  after trigger partner onchange on sale order creation

@ForgeFlow

### CODE REVIEW

File: repair_sale_order/models/repair_order.py [hunk 1]
```diff
@@ -62,7 +62,12 @@ def action_create_sale_order(self):
             sale_order_data = rec._get_sale_order_data()
             sale_order = order_model.create(sale_order_data)
             orders |= sale_order
+            partner_shipping_id = False
```
Comment (MiquelRForgeFlow) [inline repair_sale_order/models/repair_order.py:66]: why not do the same with partner_invoice_id?

Comment (ChrisOForgeFlow) [reply repair_sale_order/models/repair_order.py:66]: You're right, I added same behavior, thank you

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Review by mariadforgeflow — APPROVED** [review]
LGTM!

**Comment by grindtildeath** [comment]
While working on this module, would be good if you can add a test there, because the existing one doesn't bring anything: https://github.com/OCA/manufacture/blob/14.0/repair_sale_order/tests/test_repair_sale_order.py#L12

**Pull request merged**